README.md

```diff
@@ -0,0 +1,41 @@
+# opcua-client-ruby
+
+Incomplete OPC-UA client library for Ruby. Wraps open62541: <https://open62541.org>.
+
+## Installation
+
+Add it to your Gemfile:
+
+```ruby
+gem 'opcua_client'
+```
+
+## Usage
+
+```ruby
+require 'opcua_client'
+client = OPCUAClient::Client.new
+begin
+  client.connect("opc.tcp://127.0.0.1:4840")
```

Comment:
would be great if it supported block `client.connect(..) do ... end` (with auto disconnect inside #connect)